lib/src/main/proto/bio/terra/tanagra/underlay/mapping.proto

```diff
@@ -72,8 +72,23 @@ message RelationshipMapping {
     ColumnId foreign_key = 2;
   }
 
+  // A mapping where a relationship is navigated by a separate intermediate table.
```

Comment:
How does this interact with foreign key mappings? Should the relationships of entity1 <-> intermediate, and/or entity2 <-> intermediate exist? Not exist? Doesn't matter?

Reply:
There is no intermediate entity, just an intermediate table, so there should only be one relationship: entity 1 <-> entity 2.